tests/beta/test_default_tensor_tf.py

```diff
@@ -758,3 +758,31 @@ def circuit(x, w=None):
 
         with pytest.raises(ValueError, match="Device default.tensor.tf only supports diff_method='backprop' when using the tf interface"):
             res = cost_raising_error(params)
+
+
+class TestWiresIntegration:
+    """Test that the device integrates with PennyLane's wire management."""
+
+    def make_circuit_expval(self, wires):
+        """Factory for a qnode returning expvals using arbitrary wire labels."""
+        dev = qml.device("default.tensor.tf", wires=wires)
+
+        @qml.qnode(dev)
```

Comment:
Good point!

Reply:
But the other tests in that file seem to not specify the interface. @co9olguy , do you know why? Can the default.tensor.tf device be used with any interface qnode?

Reply:
@mariaschuld yep it can be used with any interface. If used with autograd or pytorch, it defaults to parameter-shift, if used with tf, it uses backprop (same as `default.qubit.tf`).